src/cartesian/Bar.js

```diff
@@ -184,6 +186,58 @@ class Bar extends Component {
     return labelItem;
   }
 
+  renderErrorbars() {
+    const { isAnimationActive } = this.props;
+    if (isAnimationActive && !this.state.isAnimationFinished) { return null; }
+
+    const { data, errorBarFill, errorBarStrokeWidth } = this.props;
+    const errorKey = this.props.dataKey + 'Error';
+    const errorBars = data.map((entry, i) => {
+      let xMid, yMid, xMin, yMin, xMax, yMax, scale = 0;
+      let coordsTop, coordsMid, coordsBot = {};
+      let errorVal = entry[errorKey];
+
+      if (errorVal) {
+        if (this.props.layout === 'vertical') {
+          scale = entry.width / entry.value; 
+          xMid = entry.x + entry.width;
+          yMid = entry.y + entry.height / 2;  
+          xMin = xMid - errorVal / 2 * scale;
+          yMin = yMid + entry.height / 5;
+          xMax = xMid + errorVal / 2 * scale;
+          yMax = yMid - entry.height / 5;
+          coordsTop = { x1: xMax, y1: yMin, x2: xMax, y2: yMax };
```

Comment:
The value of `errorValue` will effect `xAxisMap` here, otherwise some error bars may be cut.